recipes-bsp/device-tree/files/tepzu.dts

```diff
@@ -0,0 +1,370 @@
+			linux,code = <116>; /* KEY_POWER */
+		};
+		s1 {
+			label = "s1";
+			gpios = <&gpio 40 GPIO_ACTIVE_LOW>;
+			linux,code = <28>; /* KEY_ENTER */
+		};
+	};
+
+	leds-mio {
+		compatible = "gpio-leds";
+		leds-mio36 {
+			label = "mioled36:red";
+			/* LED connected from 3v3 to 1v8 IO, must use open drain output */
+			gpios = <&gpio 36 (GPIO_ACTIVE_LOW | GPIO_OPEN_DRAIN)>;
+			linux,default-trigger = "mmc0"; /* eMMC activity */
+		};
+	};
+
+	leds-gpioex {
+		compatible = "gpio-leds";
+		leds-eth2 {
+			label = "ethled2:green";
+			gpios = <&tep_gpioex10 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ff0d0000.ethernet-ffffffff:01:link";
+		};
+	};
+
+	/* Regulator for the 5V USB VBUS power line. */
+	reg_usb0_vbus: regulator_usb {
+		compatible = "regulator-fixed";
+		regulator-name = "usb0-vbus";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		gpio = <&tep_gpioex10 6 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+	};
+};
+
+/* SD1 */
+&sdhci1 {
+	status = "okay";
+	bus-width = <4>;
+	xlnx,mio_bank = <1>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_sdhci1_default>;
+	/* Use GPIO based card detect */
+	cd-gpios = <&gpio 45 GPIO_ACTIVE_LOW>;
+	no-1-8-v;
+	disable-wp;
+};
+
+&i2c0 {
+	status = "okay";
+	clock-frequency = <400000>;
+
+	/* I2C mux */
+	i2cmux: i2cswitch@70 {
+		compatible = "nxp,pca9544";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x70>;
+
+		i2cmux_0: i2c@0 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0>;
+
+			/* GPIO expander U8 */
+			tep_gpioex8: mcp23009@20 {
+				compatible = "microchip,mcp23008";
+				reg = <0x20>;
+				vcc-supply = <&reg_3v3_miami>;
+				gpio-line-names = "", "PEWAKE0", "PEWAKE1", "USB_OTG_OC_N", "nPRSNT_M2C", "PG_M2C", "", "";
+				gpio-controller;
+				#gpio-cells = <2>;
+				interrupt-controller;
+				#interrupt-cells = <2>;
+				interrupt-parent = <&gpio>;
+				interrupts = <32 IRQ_TYPE_LEVEL_LOW>;
+				pinctrl-names = "default";
+				pinctrl-0 = <&gpio8pullups>;
+
+				gpio8pullups: pinmux {
+					pins =	"gpio0", "gpio1", "gpio2", "gpio3",
+						"gpio4", "gpio5", "gpio6", "gpio7";
+					bias-pull-up;
+				};
+			};
+
+			/* GPIO expander U10 */
+			tep_gpioex10: mcp23009@23 {
+				compatible = "microchip,mcp23008";
+				reg = <0x23>;
+				vcc-supply = <&reg_3v3_miami>;
+				gpio-line-names = "ETH_LED2", "GPIO_SW0", "GPIO_SW1", "HRESET", "", "USB_nRST", "USB2_5V_EN", "";
+				gpio-controller;
+				#gpio-cells = <2>;
+				interrupt-controller;
+				#interrupt-cells = <2>;
+				interrupt-parent = <&gpio>;
+				interrupts = <33 IRQ_TYPE_LEVEL_LOW>;
+				pinctrl-names = "default";
+				pinctrl-0 = <&gpio10pullups>;
+
+				gpio10pullups: pinmux {
+					pins =	"gpio0", "gpio1", "gpio2", "gpio3",
+						"gpio4", "gpio5", "gpio6", "gpio7";
+					bias-pull-up;
+				};
+
+				hog-hreset {
+					gpio-hog;
+					gpios = <3 0>;
+					output-low;
+					line-name = "HRESET";
+				};
+			};
+
+			/* Also connected to PCIe switch */
```

Comment:
What does this comment belong to?

Reply:
To the I2C bus... there's a PCIe switch but no driver instantiated for it